docs/source/daml2ts/index.rst

```diff
@@ -0,0 +1,291 @@
++-------------------+------------------+----------------------------------+---------------------------------------------------------------------+
+| ``Numeric``       | ``Numeric n``    | ``string``                       | ``Numeric: (n: number) => Serializable<Numeric>``                   |
++-------------------+------------------+----------------------------------+---------------------------------------------------------------------+
+| ``Text``          | ``Text``         | ``string``                       | ``Text: Serializable<Text>``                                        |
++-------------------+------------------+----------------------------------+---------------------------------------------------------------------+
+| ``Time``          | ``Time``         | ``string``                       | ``Time: Serializable<Time>``                                        |
++-------------------+------------------+----------------------------------+---------------------------------------------------------------------+
+| ``Party``         | ``Party``        | ``string``                       | ``Party: Serializable<Party>``                                      |
++-------------------+------------------+----------------------------------+---------------------------------------------------------------------+
+| ``List<T>``       | ``[t]``          | ``T[]``                          | ``List: (t: Serializable<T>) => Serializable<List<T>>``             |
++-------------------+------------------+----------------------------------+---------------------------------------------------------------------+
+| ``Date``          | ``Date``         | ``string``                       | ``Date: Serializable<Date>``                                        |
++-------------------+------------------+----------------------------------+---------------------------------------------------------------------+
+| ``ContractId<T>`` | ``ContractId t`` | ``string``                       | ``ContractId: (t: Serializable<T>) => Serializable<ContractId<T>>`` |
++-------------------+------------------+----------------------------------+---------------------------------------------------------------------+
+| ``Optional<T>``   | ``Optional t``   | ``null | (null extends T ?``     | ``Optional: (t: Serializable<T>) => Serializable<Optional<T>>``     |
+|                   |                  | ``[] | [Exclude<T, null>] : T)`` |                                                                     |
++-------------------+------------------+----------------------------------+---------------------------------------------------------------------+
+| ``TextMap<T>``    | ``TextMap t``    | ``{ [key: string]: T }``         | ``TextMap: (t: Serializable<T>) => Serializable<TextMap<T>>``       |
++-------------------+------------------+----------------------------------+---------------------------------------------------------------------+
+
+DAML to Typescript mappings
```

Comment:
```suggestion
DAML to TypeScript mappings
```
That’s how typescript is capitalized on the official web page https://www.typescriptlang.org/ so we should follow that. There are quite a few occurences here but I’ll refrain from marking all of them since it’s just a search and replace anyway to fix all of them.

Reply:
Thank-you. Will fix!